sched/scheduler/jobStatus.go

```diff
@@ -1,83 +0,0 @@
-package scheduler
```

Comment:
I'd suggest keeping this file. That sounds crazy and like duplication, so let me explain:

We'll want to test the status of jobs in scheduler code. It'll be easier to do that if it's a POGO struct instead of being a Thrift object with all of its weirdness.

Then scheduler code does saga -> jobStatus, and server does jobStatus -> thrift.

This keeps thrift at the edges of the system.

(Though it's not important for this review, it's a shame to delete this nice code you've already written!)